lua/telekasten.lua

```diff
@@ -1975,428 +2069,444 @@ local function FollowLink(opts)
+                    find_files_sorted({
+                        prompt_title = "Follow link to note...",
+                        cwd = pinfo.root_dir,
+                        default_text = title,
+                        find_command = M.Cfg.find_command,
+                        attach_mappings = function(_, map)
+                            actions.select_default:replace(
+                                picker_actions.select_default
+                            )
+                            map("i", "<c-y>", picker_actions.yank_link(opts))
+                            map("i", "<c-i>", picker_actions.paste_link(opts))
+                            map("n", "<c-y>", picker_actions.yank_link(opts))
+                            map("n", "<c-i>", picker_actions.paste_link(opts))
+                            map("n", "<c-c>", picker_actions.close(opts))
+                            map("n", "<esc>", picker_actions.close(opts))
+                            return true
+                        end,
+                        sort = M.Cfg.sort,
+                    })
+                end
 
-        -- check if fname exists anywhere
-        local pinfo = Pinfo:new({ title = title })
-        if
-            (pinfo.fexists ~= true)
-            and (
-                (opts.follow_creates_nonexisting == true)
-                or M.Cfg.follow_creates_nonexisting == true
-            )
-        then
-            if opts.template_handling == "always_ask" then
-                return on_create_with_template(opts, title)
-            end
+                if
+                    (pinfo.fexists ~= true)
+                    and (
+                        (opts.follow_creates_nonexisting == true)
+                        or M.Cfg.follow_creates_nonexisting == true
+                    )
+                then
+                    if opts.template_handling == "always_ask" then
+                        return on_create_with_template(opts, title)
+                    end
 
-            if #pinfo.filepath > 0 then
-                local uuid = fileutils.new_uuid(uuid_type)
-                create_note_from_template(
-                    title,
-                    uuid,
-                    pinfo.filepath,
-                    pinfo.template,
-                    pinfo.calendar_info
-                )
-                opts.erase = true
-                opts.erase_file = pinfo.filepath
-            end
-        end
+                    if #pinfo.filepath > 0 then
+                        local uuid = fileutils.new_uuid(uuid_type)
+                        create_note_from_template(
+                            title,
+                            uuid,
+                            pinfo.filepath,
+                            pinfo.template,
+                            pinfo.calendar_info,
+                            function()
+                                opts.erase = true
+                                opts.erase_file = fname
```

Comment:
Stylua complains here that fname is not defined properly. Why not keep `pinfo.filepath`?

Reply:
hm, looks like this was an oversight on my part

Reply:
updated